.gitignore

```diff
@@ -57,3 +57,4 @@ cli_build
 cli_output
 *.js.map
 ts-dist
+packages/electron/build-scripts/porting-assistant-config.dev.json
```

Comment:
why you ignore this file

Reply:
I did not want to overwrite the version in the develop branch, which is missing the endpoint info. Should I remove it from gitignore?

Reply:
plz remove, once you merge it. It would overwrite the main branch

Reply:
removed